kedro-datasets/test_requirements.txt

```diff
@@ -24,13 +24,15 @@ lxml~=4.6
 matplotlib>=3.0.3, <3.4; python_version < '3.10' # 3.4.0 breaks holoviews
 matplotlib>=3.5, <3.6; python_version == '3.10'
 memory_profiler>=0.50.0, <1.0
+mlflow~=1.30.0; python_version=='3.7'
+mlflow~=2.2.1; python_version>='3.8'
```

Comment:
Where are these dependencies coming from? I don't see `mlflow` is imported

Reply:
True, good spot. These are for a set of MLflow Datasets that are highly related to this one and will get their own PR soon. I'll remove these for now.